src/app/static/src/js/components/runReport/runReportTabs.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <div id="report-log-container" class="mt-3">
+        <div class="row">
+            <div class="col-12 col-md-4 col-xl-3">
+                <div class="sidebar pb-4 pb-md-0">
+                    <nav class="pl-0 pr-0 pr-md-4 navbar navbar-light">
+                        <button type="button" class="d-md-none navbar-toggler" data-toggle="collapse"
+                                data-target="#sidebar">
+                            <span class="navbar-toggler-icon"></span>
+                        </button>
+                        <div class="d-md-block mt-4 mt-md-0 collapse navbar-collapse" id="sidebar">
+                            <ul class="nav flex-column list-unstyled mb-0">
+                                <li class="nav-item">
+                                    <a id="run-link" class="nav-link active" data-toggle="tab" role="tab" @click="switchTab('runReport')">Run a report</a>
```

Comment:
Best to add an `href="#"`here - the usual link hover pointer doesn't display on Firefox without an `href`

Reply:
Chrome too..